decidim-core/lib/decidim/api/types/badge_score_type.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Core
+    class BadgeScoreType < Decidim::Api::Types::BaseObject
+      description "An user badge score"
+
+      field :description, GraphQL::Types::String, "The badge's description", null: false
+      field :image, GraphQL::Types::String, "The badge's image", null: false
+      field :level, GraphQL::Types::Int, "The badge's level", null: false
+      field :name, GraphQL::Types::String, "The badge's name", null: false, method: :badge_name
+      field :score, GraphQL::Types::Int, "The badge's score", null: false
```

Comment:
I see that we're generally not consistent in the usage of the formula used for the descriptions. I propose to at least be consistent on this PR

```suggestion
      field :description, GraphQL::Types::String, "The description of this badge", null: false
      field :image, GraphQL::Types::String, "The image of this badge", null: false
      field :level, GraphQL::Types::Int, "The level of this badge", null: false
      field :name, GraphQL::Types::String, "The name of this badge", null: false, method: :badge_name
      field :score, GraphQL::Types::Int, "The score of this badge", null: false
```